src/StateConfig.php

```diff
@@ -80,6 +80,25 @@ public function allowTransitions(array $transitions): StateConfig
         return $this;
     }
 
+    /**
+    * @param \Illuminate\Database\Eloquent\Model $model
+    * @param string $fromClass
+    */
+    public function allowedTransitionsFrom(Model $model, string $fromClass)
+    {
+        $allowedTransitionsFrom = [];
+
+        foreach ($this->allowedTransitions as $allowedTransition => $value) {
+            list($from, $to) = explode('-', $allowedTransition);
+
+            if ($from === $fromClass) {
+                $allowedTransitionsFrom[] = get_class($this->stateClass::make($to, $model));
```

Comment:
I'd use `$this->stateClass::getMorphClass()` here, instead of directly `get_class`. This means that the developer using this code will need to do this mapping in their code, but it adds more flexibility, and on top of that you don't have to pass `$model` here. It would look something like this:

```php
public function allowedTransitionsFrom(Model $model, string $fromClass)
{
    $allowedTransitionsFrom = [];

    foreach ($this->allowedTransitions as $allowedTransition => $value) {
        [$from, $to] = explode('-', $allowedTransition);
        
        if ($from !== $fromClass) {
            continue;
        }

        $allowedTransitionsFrom[] = $to::getMorphClass();
    }
    
    return $allowedTransitionsFrom;
}
```

Also note the early continue statement to keep the code better readable.